src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -489,10 +500,15 @@ export default {
 	background-color: var(--color-main-background);
 	border-radius: calc($clickable-area / 2);
 	box-shadow: 0 0 4px 0 var(--color-box-shadow);
+	height: 44px;
 
 	& h6 {
 		margin-left: auto;
 	}
+
+	&--last {
+		bottom: 36px;
```

Comment:
This feels very weird if for one message the buttons are at another position. Can't we instead add more offset or something?

I replaced it therefor with 36px from bottom and now it looks/overlaps like with any other message
![Bildschirmfoto vom 2022-08-02 15-48-20](https://user-images.githubusercontent.com/213943/182390972-1952c210-2a70-457f-a311-259459646edc.png)